docs/build/create-project-with-linkingrule/index.md

```diff
@@ -0,0 +1,322 @@
+Sources of information are LinkedMDB, the linked data version of the IMDb movie database, and DBpedia, the linked data version of Wikipedia.
+Both files contain a list of movies with their names, titles, release dates, and an internal ID for each movie.
+The task is to link the two datasets to find out which movie in DBpedia corresponds to which movie in LinkedMDB.
+
+## Sample material
+
+The following material is used in this tutorial, you should download the files and have them at hand throughout the tutorial:
+
+-   [dbpedia.csv](dbpedia.csv)
+
+    !!! info
+
+        ![](preview-dbpedia.png){ width="75%"}
+
+-   [linkedmdb.csv](linkedmdb.csv)
+
+    !!! info
+
+        ![](preview-linkedmdb.png){ width="75%"}
+
+## 1. Create a new project
+
+-   Once logged in to Corporate memory on the right side click on **create.**
+
+    ![image](22.2-create-project.png){ width="75%"}
+
+    The screen gives an overview of the different projects created in your installation of DataIntegration.
+    It shows the item type on the left side like a project, workflow, dataset, transform, linking, and task.
+
+-   Click on the **project** on the left side in the item type and double-click on the **project** in the centre.
+
+    ![image](22.2-click-on-project.png){ width="75%"}
+
+-   Type the project name as **Movie links** in the title field and click on **create.**
+
+    ![image](22.2-type-movielink.png){ width="75%"}
+
+!!! success "Step Result"
+
+    The project was created with the name "Movie links," which is displayed in the page top left corner.
+
+    ![image](22.2-project-created.png){ width="75%"}
+
+## 2. Upload the files
+
+-   Click on **add file** on the right side of the page.
+
+    ![image](22.2-click-on-add-files.png){ width="75%"}
+
+-   Click on **drop files here** select the file and click on **open**.
+
+    ![image](22.2-upload-files.png){ width="75%"}
+
+-   Once the file is successfully uploaded click on **close.**
+
+    ![image](22.2-files-uploaded.png){ width="75%"}
+
+-   Repeat the same step to add the file select the next second file and upload it.
+
+!!! success "Step Result"
+
+    The files are uploaded to the project and reflect on the right side as shown below.
+
+    ![image](22.2-files-added.png){ width="75%"}
+
+## 3. Create datasets
+
+-   Click on **create.**
+
+    ![image](22.2-click-on-create.png){ width="75%"}
+
+-   Click on the **dataset** on the left side in the item type and double-click on the **CSV file** in the centre.
+
+    ![image](22.2-click-on-dataset.png){ width="75%"}
+
+-   Type the file name as **dbpedia** in the label field.
+
+    ![image](22.2-type-dbpedia.png){ width="75%"}
+
+-   Tick the option **select file from project** and select the file from the drop-down list of files available in the project.
+
+    ![image](22.2-select-dbpedia-file.png){ width="75%"}
+
+-   Click on **create**.
+
+    ![image](22.2-click-on-create1.png){ width="75%"}
```

Comment:
The image is cropped wrongly so it looks ugly.

Reply:
I mean that the buttons are not completely seen and that there is a black line on the right.